### PR TITLE
Help Page

### LINKED ISSUE #56: Help Button Documentation
Make a `/help` page to display documentation. If there's time, use anchors for page-specific help.

### PR DESCRIPTION
**Pull request by blackjack26**
- Added template for help page
- Table of Contents
- Permission / Context based

Fixes #56

### CODE REVIEW

**Comment by vmjoseph** [comment]
Looks great! Just making sure this isn't on my end only, but my links to items under the "Class Activity" list (Survey and Attendance) seem not to be working.

**Comment by blackjack26** [comment]
@vmjoseph Added links to help for the attendance and survey pages

File: views/help.php [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+                <b>Main Content</b> - This is the largest portion of the page which will display all of the content
+                for each page.
+            </li>
+        </ul>
+
+        <?php if (hasRole(Role::User)) { ?>
+            <h4 id="agency">Agency Requests</h4>
+            <div>
+                <p>
+                    The Agency Requests page provides a search engine for the program’s participants, where a user can
```
Comment (ghost) [inline views/help.php]: For consistency I think "a user" can be replaced with "you". You use "you" in the other help sections.

File: views/help.php [hunk 2]
```diff
@@ -0,0 +1,591 @@
...
+                        <li>Amount of participants served</li>
+                        <li>Participant ethnicity</li>
+                        <li>Amount of participants served organized by their home zip code</li>
+                    </ul>
+                </div>
+                <h5 id="reports-quarterly">Quarterly</h5>
+                <div>
+                    <p>
+                        The Quarterly Reports page provides you with participant totals and survey results for the
+                        Quarter chosen. Participant totals show the amount to unduplicated Adults and Children served
```
Comment (ghost) [inline views/help.php]: `totals show the amount to unduplicated` -> `totals show the amount of unduplicated`

File: views/help.php [hunk 3]
```diff
@@ -0,0 +1,591 @@
...
+                    </p>
+                </div>
+            <?php } ?>
+
+            <?php if (hasRole(Role::Admin)) { ?>
+                <h4 id="user-manage">User Management</h4>
+                <div>
+                    <p>
+                        The User Management feature allows administrators and super administrators the ability to assign
+                        roles to other users. This would is the place where new employee users would be assigned to
```
Comment (ghost) [inline views/help.php]: `This would is` -> `This would be`

File: views/help.php [hunk 4]
```diff
@@ -0,0 +1,591 @@
...
+                </p>
+                <p>
+                    To update survey data click the "Update Surveys" button located on the bottom of the page. The time
+                    that the surveys were last updated is located above this button. Only surveys before this time will
+                    be displayed in the results.
+                </p>
+            </div>
+        <?php } ?>
+
+<!--        <h4 id="faq">FAQ</h4>-->
```
Comment (ghost) [inline views/help.php:513]: Going to make an issue to collect these.

**Comment by blackjack26** [comment]
@evanmena Also most of these I copied verbatim from the documentation files so some of these errors are present in that documentation as well. Thanks for finding them 😄

**Comment by ghost** [comment]
@blackjack26 no problem, maybe send a note to the documentation owners 🗡

**Review by vmjoseph — APPROVED** [review]
I think we can go ahead and merge this for now. Reach out to the appropriate parties when possible to make the smaller changes.

**Comment by blackjack26** [comment]
@evanmena I've made those grammar changes you've pointed out. I think we can add in FAQ stuff later as an extra bonus (parking lot).

**Comment by ghost** [comment]
@blackjack26 sure thing, that's why I made a separate issue!

**Review by ghost — APPROVED** [review]

**Pull request merged**